types/change-case-object/index.d.ts

```diff
@@ -0,0 +1,28 @@
+// Type definitions for change-case-object 2.0
+// Project: https://github.com/BinaryThumb/change-case-object#readme
+// Definitions by: James Chao <https://github.com/jameschao>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+export function camelCase(s: string): string;
+export function camelCase<T>(arr: ReadonlyArray<T>): ReadonlyArray<T>;
```

Comment:
Taking a read-only array is a good idea. But why indicate this for an array that is created each time this function is called?

```suggestion
export function camelCase<T>(arr: ReadonlyArray<T>): T[];
```

Reply:
Good catch @demensky. That was unintentional.